openlibrary/i18n/messages.pot

```diff
@@ -2556,6 +2556,10 @@ msgstr ""
 msgid "Suggesting Edits"
 msgstr ""
 
+#: nav_foot.html:51
+msgid "Change Website Language"
+msgstr ""
+
```

Comment:
These are automatically generated later; let's revert the changes to this file; it looks like the file permissions also changed somehow?

Reply:
Done.